libs/shared/sai-editor/src/sai-editor.component.ts

```diff
@@ -87,7 +87,7 @@ import { EditorButtonsComponent, QueryOutputComponent } from '@keira/shared/base
   ],
 })
 export class SaiEditorComponent extends MultiRowEditorComponent<SmartScripts> implements OnInit {
-  override readonly editorService = inject(SaiEditorService);
+  public override readonly editorService = inject(SaiEditorService);
```

Comment:
we should define a codestyle about writing `public`, I personally don't write the code that is unnecessary, but I see that there is no consistency in Keira3 @FrancescoBorzi